compliance/containers/cis-docker-1.2.0.yaml

```diff
@@ -5,103 +5,103 @@ framework: cis-docker
 version: 1.2.0
 rules:
   - id: cis-docker-1.2.0-1.2.1
-    description: '[CIS Docker] Ensure a separate partition for containers has been created'
+    description: '[CIS Docker] A separate partition for containers has been created'
```

Comment:
I think previous descriptions come from the CIS benchmark document.

Reply:
Yes initially, the docs team has done a big pass afterwards to normalize titles and descriptions across all rules.

Reply:
ok, 
maybe it will not ease the possibility to update rules from the CIS benchmark document